README.md

```diff
@@ -3,4 +3,39 @@
 # external-dns-app
 Helm chart for external-dns service running in guest clusters
 
-WARNING: This Helm chart works only with Azure at the moment.
+## Installing the Chart
+
+To install the chart locally:
+
+```bash
+$ git clone https://github.com/giantswarm/external-dns-app.git
+$ cd external-dns-app
+$ helm install helm/external-dns-app
+```
+
+Provide a custom `values.yaml`:
+
+```bash
+$ helm install external-dns-app -f values.yaml
+```
+
+Deployment to Tenant Clusters is handled by [app-operator](https://github.com/giantswarm/app-operator).
+
+## Configuration
+
+Configuration options are documented in [Configuration.md](helm/external-dns-app/Configuration.md) document.
+
+## Release Process
```

Comment:
Is this something we have for all apps? I would rather to have it just one place in common, so easy to maintain.

Reply:
@pipo02mix Yes I've been adding this README format to all the new app repos.

I see your point about making it DRY with a link but the steps are very generic and unlikely to change.

Reply:
ok good to know :)